### ISSUE TITLE
web searches full and simple all default search urls have disappeared

### ISSUE THREAD

**Issue by liquidmetal2** [quicksilver/Quicksilver#2454]
Just noticed today. all of them have disappeared so not able to use QS initiate a search. 
Edit: oh btw, I'm on Mojave public beta 6. Just struck me maybe that's why the search URLs have gone?

![image](https://user-images.githubusercontent.com/2692241/44248326-83880380-a207-11e8-8142-1e0bedf245fb.jpeg)

![image](https://user-images.githubusercontent.com/2692241/44248342-98649700-a207-11e8-9980-7be85268053f.jpeg)

![image](https://user-images.githubusercontent.com/2692241/44248360-b500cf00-a207-11e8-96cc-5164547a3558.jpeg)

**Comment by skurfer** [comment]
I just tested under Mojave and the entries populate as expected.

Our hosting provider has been dealing with hardware issues. The most likely cause is that the site was down last time your copy of QS tried to update those entries. If you select them and hit the circular “refresh” arrow at the bottom of the window, they should come back.

**Comment by liquidmetal2** [comment]
Thanks. After I posted that problem on the site, I reinstalled the web search plugins along with some other plugins e.g. 1 password and couple others. With that, QS started crashing or wouldn’t even start. Trashed the app, removed all folders, plists etc. Reinstalled the QS app 1.5.9 and same thing. Today, I again trashed everything of the app, and instead installed the version previous v.1.5.6 and that is working fine (no crashes until now)!!

Have submitted multiple crash reports (the auto ones that pop up)




Thank you,

Ravindra Holalkere

A developed country is not where the poor have cars, it's where the rich use public transport 
												
															— (Unknown) Mayor of Bogota. 


The greatest tragedy in mankind's entire history may be the hijacking of morality by religion. 

														    	—Arthur C Clarke
